src/common/OurUsers.mjs

```diff
@@ -121,6 +121,12 @@ var companies = [
     url: "https://camelohq.com",
     path: "/static/lp/camelo.svg",
     [Symbol.for("name")]: "Logo"
+  },
+  {
+    name: "DevIT Jobs",
```

Comment:
was this an intentional change? if so, could you factor that into its own PR?

Reply:
No this change was unintentional, not sure how that happened though 😅.